### PR TITLE
Upgrade to ES6 structure

### PR DESCRIPTION
**Pull request by teckim**
In this pull request:

- Split the code into multiple modules.
- Use ES6
- Use Luxon module

### CODE REVIEW

File: src/modules/books-list.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    this.#persist();
+    this.#render();
+  }
+
+  remove(id) {
+    if (!id) return;
+
+    this.books = this.books.filter((book) => book.id !== id);
+    this.#persist();
+  }
```
Comment (AnselemOdims) [inline src/modules/books-list.js:27]: - [ ] Please kindly ensure that the book is actually removed from the localStorage. At the moment, it removes it from the DOM but when the page is refreshed it reappears.

File: src/modules/books-list.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+const booksListEl = document.querySelector('.books-list');
+
+function uid() {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+}
```
Comment (AnselemOdims) [inline src/modules/books-list.js]: - [ ] Please kindly stick to using ES6 arrow function.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @teckim , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/books-list.js (hunk 1 above)
Comment (teckim) [reply src/modules/books-list.js:27]: Hi @AnselemOdims, actually it's removing it from the localStorage, this.#persist() does that job.

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @teckim 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**